.github/workflows/km-ci-workflow.yaml

```diff
@@ -540,8 +540,8 @@ jobs:
       uses: github/codeql-action/init@v1
       with:
         languages: ${{ matrix.language }}
-        # TODO: Enable enhanced security checks.
-        # queries: security-extended
+        # enhanced security checks.
+        queries: security-extended
```

Comment:
Maybe move this up so that we don't wait for all the tests to complete before it fails on the QL check